frontend/components/Dashboard/BoardList/CardBody.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import Link from "next/link";
+import {
+  HoverCardTrigger,
+  HoverCardContent,
+  HoverCardArrow,
+  HoverCardRoot,
+} from "../../Primitives/HoverCard";
+import { BoardType } from "../../../types/board";
+import { ColumnType } from "../../../types/column";
+import Card from "../../Primitives/Card";
+import Flex from "../../Primitives/Flex";
+import CardHeader from "./CardHeader";
+import { styled } from "../../../stitches.config";
+import Shape from "../../Primitives/Shape";
+import Text from "../../Primitives/Text";
+import InputTitle from "../../Title/InputTitle";
+
+const Circle = styled(Shape, {
+  size: "$32",
+});
+
+const HandleCols = React.memo<{ columns: ColumnType[] }>(({ columns }) => {
+  return (
+    <>
+      {columns.map((column) => {
+        const trigger = (
+          <Circle
+            variant="circle"
+            align="center"
+            justify="center"
+            css={{
+              color: "white",
+              backgroundColor: column.color === undefined ? "gray" : column.color,
+            }}
+          >
+            {column.cardsOrder.length}
+          </Circle>
+        );
+        return (
+          <HoverCardRoot key={column._id}>
+            <HoverCardTrigger asChild>{trigger}</HoverCardTrigger>
+            <HoverCardContent sideOffset={0}>
+              {column.title}
+              <HoverCardArrow />
+            </HoverCardContent>
+          </HoverCardRoot>
+        );
+      })}
+    </>
+  );
+});
+
+const CardBody: React.FC<{ board: BoardType }> = ({ board }) => {
+  const [showEditTitle, setShowEditTitle] = useState(false);
+
+  return (
+    <Link key={board._id} href={`boards/${board._id}`}>
+      <Card
+        css={{ size: "$220", px: "$20", py: "$8", pointerEvents: showEditTitle ? "none" : "all" }}
+        direction="column"
+        align="center"
+        justify="start"
+        gap="26"
+        radius="40"
+        interactive="clickable"
+      >
+        <CardHeader board={board} isEditing={showEditTitle} onClickEdit={setShowEditTitle} />
+        {showEditTitle ? (
+          <InputTitle board={board} onClickEdit={setShowEditTitle} isBoardPage={false} />
+        ) : (
+          <Text size="20" fontWeight="bold">
+            {board.title}
+          </Text>
+        )}
+        <Text>{board.creationDate?.slice(0, 10)}</Text>
+        {board.columns && (
+          <Flex gap="8">
+            <HandleCols columns={board.columns} />
+          </Flex>
+        )}
```

Comment:
We should always make sure we have a boolean when using logic operators for conditional rendering. (ref: https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)

```suggestion
        {!!board.columns && (
          <Flex gap="8">
            <HandleCols columns={board.columns} />
          </Flex>
        )}
```

Reply:
thanks for the tip!!